public/app.js

```diff
@@ -37,14 +53,22 @@ class Controller {
             DataController.detachSnapshotListener();
             console.log(e.target.innerText);
             let project = e.target.innerText;
-            // Add the graphs container to the UI
-            UIController.addGraphs(project);
-            // Update and show the Graphs
-            let unsubscribeFunc = DataController.watchProjectTrafficData(
-                project,
-                GraphController.updateGraphs
-            );
-            DataController.registerSnapshotListener(unsubscribeFunc);
+            location.hash = project;
+            Controller.displaySelectedProject(project);
+        }
+    }
+
+    static deepLinktoPage(activeProjectsData) {
```

Comment:
This name is confusing, I can't tell whether you're setting a deep link or reading a deep link.

How about displayDeepLinkedTrafficPage? 

It might help readability to pass in the link's hash component as a variable too.

Reply:
noted